spacing.go

```diff
@@ -1,14 +1,15 @@
-// Package clio contains helpers for printing CLI output messages.
 package clio
 
-import "strings"
+import (
+	"strings"
+)
 
-// NewLine prints a newline to clio.OutputWriter
+// NewLine prints a newline to clio.ErrorWriter
 func NewLine() {
-	stdout.Println()
+	stderrlog.Println()
```

Comment:
@JoshuaWilkes I changed this to print to stderr rather than stdout. Looking at https://github.com/common-fate/granted/pull/292 I see that you've overridden stdout to print to stderr in all cases, I feel like this should default to stderr.

Maybe we should remove `clio.Log` altogether from this package and have the package *only* print to stderr?